docs/source/databrowser/cli.rst

```diff
@@ -108,10 +108,10 @@ There are many more options for defining a value for a given key:
 |                                                 | attribute1 AND value2  |
 |                                                 | for attribute2         |
 +-------------------------------------------------+------------------------+
-| ``attribute_not_=value``                        | Search for files NOT   |
+| ``attribute=-value`` ``attribute=not value``    | Search for files NOT   |
 |                                                 | containing value       |
 +-------------------------------------------------+------------------------+
-| ``attribute_not_=value1 attribute_not_=value2`` | Search for files       |
+| ``attribute=-value1 attribute=not value2``      | Search for files       |
```

Comment:
sorry for this dumb question, but has been the cli (freva core) changed so that is reflected? or does it go independently?

Reply:
That's a very good question. No, it hasn't yet. I think we should switch to using the databrowser api also in the old freva core.